react/features/base/config/functions.web.ts

```diff
@@ -102,6 +102,20 @@ export function _setDeeplinkingDefaults(deeplinking: IDeeplinkingConfig) {
 
     desktop.appName = desktop.appName || 'Jitsi Meet';
     desktop.appScheme = desktop.appScheme || 'jitsi-meet';
+    if (desktop.download) {
```

Comment:
Can you pl simplify this so we don't need to duplicate the URLs? Something like:

```
desktop.download = desktop.download || {};
 desktop.download.windows = desktop.download.windows
            || 'https://github.com/jitsi/jitsi-meet-electron/releases/latest/download/jitsi-meet.exe';
        desktop.download.macos = desktop.download.macos
            || 'https://github.com/jitsi/jitsi-meet-electron/releases/latest/download/jitsi-meet.dmg';
        desktop.download.linux = desktop.download.linux
            || 'https://github.com/jitsi/jitsi-meet-electron/releases/latest/download/jitsi-meet-x86_64.AppImage';
```